spec/book_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../book'
```

Comment:
- [ ] It would be great if you test all the methods which are in the book class to follow the project requirements 👍 because you have some custome methods such as `add_rental` etc 🤔